projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt

```diff
@@ -495,6 +496,16 @@ class ProjectorServer private constructor(
       is ClientWindowsDeactivationEvent -> {
         updateWindowsState(message.windowIds, WindowEvent.WINDOW_DEACTIVATED)
       }
+      is ClientNotificationEvent -> {
+        val intellijNotificationType = when (message.notificationType) {
+          ClientNotificationType.INFORMATION -> NotificationType.INFORMATION
+          ClientNotificationType.WARNING -> NotificationType.WARNING
+          ClientNotificationType.ERROR -> NotificationType.ERROR
+        }
+
+        val notification = Notification("ProjectorClient", message.title, message.message, intellijNotificationType)
```

Comment:
This line is marked as red in IDEA: `Cannot resolve notification group id 'ProjectorClient'`. Maybe we should suppress this or solve it in some other way.

Also, what happens here if we run Projector with a plain Swing app (not IntelliJ-based)? Probably we should surround these lines that use IntelliJ classes to `runWhenIdeIsInitialized`?